Dockerfile

```diff
@@ -1,23 +1,47 @@
-FROM ubuntu:20.04
+# --------------------------------------------------
+# Base container
+# --------------------------------------------------
+FROM docker.io/ubuntu:focal AS base
 
 RUN set -e
 
-# Setup time-zone so that the build does not hang
-# on configuring the tzdata package.
-# I work in Brno, that is basically Vienna-North :)
-# (definitely its closer than Prague)
-ENV TZ=Europe/Vienna
-RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime
-RUN echo "$TZ" > /etc/timezone
-
 # Install packages
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
-RUN apt-get install -y git cmake make llvm zlib1g-dev clang g++ python3
+RUN apt-get install -yq --no-install-recommends clang emacs llvm vim
```

Comment:
Why do we need `emacs` and `vim` here? I see they are useful when playing with the image, but maybe it would be better to leave the installation of the preferred editor to the user?

Reply:
My bad, I just briefly skimmed through the documentation and read by mistake the complete opposite that these tools are pre-installed. I'll remove them.